spread.yaml

```diff
@@ -922,8 +919,14 @@ suites:
             #shellcheck source=tests/lib/pkgdb.sh
             . "$TESTSLIB"/pkgdb.sh
             distro_update_package_db
-            distro_install_package snapd qemu qemu-utils genisoimage sshpass qemu-kvm cloud-image-utils ovmf kpartx xz-utils
-            snap install --classic --channel="$UBUNTU_IMAGE_SNAP_CHANNEL" ubuntu-image
+            distro_install_package snapd qemu qemu-utils genisoimage sshpass qemu-kvm cloud-image-utils ovmf kpartx xz-utils mtools
+            # build a custom version ubuntu-image with test keys
+            (
+                export GO111MODULE=off
+                # use go get so that ubuntu-image is built with current snapd sources
+                go get github.com/canonical/ubuntu-image/cmd/ubuntu-image
```

Comment:
it would be nice to fetch a specific tag/branch/etc rather than just pulling straight from main, but not sure what is appropriate

Reply:
There are tags which correspond to ubuntu-image package versions, eg. `2.1+22.04ubuntu2`